src/compiler/sys.ts

```diff
@@ -1186,7 +1186,7 @@ namespace ts {
             let activeSession: import("inspector").Session | "stopping" | undefined;
             let profilePath = "./profile.cpuprofile";
 
-            const realpathSync = _fs.realpathSync.native ?? _fs.realpathSync;
+            const realpathSync = process.platform !== "win32" ? _fs.realpathSync.native : _fs.realpathSync;

```

Comment:
```suggestion
            const realpathSync = process.platform !== "win32" ? (_fs.realpathSync.native ?? _fs.realpathSync) : _fs.realpathSync;
```
Because `realpathSync.native` is only available on node 9.2 or higher (otherwise it can be a breaking change).